src/app/ApplicationTemplate.Tests/Framework/IntegrationTestBase.cs

```diff
@@ -27,6 +30,11 @@ public IntegrationTestBase()
 			ConfigurationSetup(ConfigureHost);
 		}
 
+		/// <summary>
+		/// Initializes different services used by the app.
+		/// Normally used for mocking services.
```

Comment:
I think this comment applies to `extraHostConfiguration` more that the method itself, right?